atmo/jobs/models.py

```diff
@@ -385,6 +385,15 @@ def spark_job_identifier(self):
     def info(self):
         return self.spark_job.cluster_provisioner.info(self.jobflow_id)
 
+    @property
+    def status_css_class(self):
+        if self.status in Cluster.ACTIVE_STATUS_LIST:
+            return 'success'
+        if self.status in Cluster.FAILED_STATUS_LIST:
+            return 'danger'
+
+        return None
```

Comment:
A few notes here:
- There are already helper properties called `Cluster.is_active` and `Cluster.is_failed` that are in line with this logic and we should add similar helpers to the `SparkJobRun` model
- There is no need to have the CSS class thing happening in the data model, it's mixing data handling with representation which will be harder to maintain on the long run, let's move this into a template tag